Islandora-CLAW/Crayfish
# Clean psr2

**GitHub Issue**: none

This is the follow up to #54 with the code fixes in it.

# What does this Pull Request do?

Does a bunch of clean-up and alters our test runner to remember it failed and tell the outside world.

# What's new?
 
* Does this change require documentation to be updated? no
* Does this change add any new dependencies? no
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? no
* Could this change impact execution of existing code? no

# How should this be tested?

Check the Travis output and ensure we are actually passing. I contemplating

Travis passes and because the code is really clean.

# Interested parties
@Islandora-CLAW/committers
